File: src/main/java/com/quid/kopring/dto/user/request/UserUpdateRequest.java
package com.quid.kopring.dto.user.request;

import lombok.AllArgsConstructor;

@AllArgsConstructor
public class UserUpdateRequest {

  private long id;
  private String name;

  public long getId() {
    return id;
  }

  public String getName() {
    return name;
  }

}

File: src/main/kotlin/com/quid/kopring/user/model/request/UserUpdateRequest.kt
package com.quid.kopring.dto.user.request;

import lombok.Builder;

public class UserUpdateRequest {

  private long id;
  private String name;

  public long getId() {
    return id;
  }

  public String getName() {
    return name;
  }

  @Builder
  public UserUpdateRequest(long id, String name) {
    this.id = id;
    this.name = name;
  }
}
